src/index.html

```diff
@@ -12,16 +12,436 @@
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__elem">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__elem">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__elem">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__elem">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__elem">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__elem">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__elem">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__elem">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div
+        class="card card--1"
+        data-qa="card"
+      >
+        <div class="image">
+          <img
+            src="/src/images/imac.jpeg"
+            alt="apple_image"
+            class="image__display image__display--settings"
+          />
+        </div>
+
+        <a
+          href="#"
+          class="card__product"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+        <section class="card__code">Product code: 195434</section>
+
+        <div class="rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="rating__review">Reviews: 5</div>
+        </div>
+
+        <div class="cash">
+          <div class="cash__name cash__name--price">Price:</div>
+          <div class="cash__sum cash__sum--value-of-price">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
```

Comment:
Ensure that the `data-qa="card-hover"` attribute is applied only to the `Buy` link inside the first card, as per the task requirements.